src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -1065,4 +1120,30 @@ private static CalendarDTO createWeekdayCalendar(String serviceId, String startD
         LOG.info(output);
         return mapper.readValue(output, CalendarDTO.class);
     }
+
+    /**
+     * Create and store a simple calendar date that modifies service on one day.
+     */
+    private static CalendarDateDTO createAndStoreCalendarDate(String serviceId, String date, int exceptionType)  throws IOException, SQLException, InvalidNamespaceException {
```

Comment:
Remove extra space before `throws`.

Reply:
Updated in https://github.com/conveyal/gtfs-lib/pull/376/commits/577ae945553f3c5f6f2a480ae4a0588b5c40e116